### PR TITLE
Create placeholder files and directories based on hca

### PR DESCRIPTION
**Pull request by jbalboni**
Addresses #2716.

This has some stripped down and placeholder files/directories based on the hca folder. It looks like there's more we could reuse from hca, but since I'm not sure exactly how much, I just brought over the minimum.

I haven't figure out how to view this app in vets-website yet, but it builds in Webpack so I'm putting it up for review in the meantime.

### CODE REVIEW

**Comment by markolson** [comment]
`options-for-select.js` looks like a _prime_ candidate for pulling into a common shared lib in this larger app/repo

**Comment by jbalboni** [comment]
@markolson Yeah, I think so.

**Comment by jbalboni** [comment]
This seems to be working now, so it's ready for review, @akainic. Wasn't sure about some of the naming or if I created a Markdown entry page in the right spot.

File: src/js/edu-benefits/utils/options-for-select.js [hunk 1]
```diff
@@ -0,0 +1,1594 @@
+const countries = [
```
Comment (akainic) [inline src/js/edu-benefits/utils/options-for-select.js]: If it looks like these can be shared across apps perhaps we can move it out of the edu-benefits directory and somewhere all apps can easily access? Doesn't have to be done in this first pass though, let's just keep in mind once the HCA is merged in and we can move things around more easily.

File: src/sass/_hca.scss [hunk 2]
```diff
@@ -0,0 +1,216 @@
+// Variables
```
Comment (akainic) [inline src/sass/_hca.scss]: Was this file not imported in the HCA migration PR? oops, good catch!

File: test/hca/components/HealthCareApp.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (akainic) [inline test/hca/components/HealthCareApp.spec.jsx]: I think this was brought in in https://github.com/department-of-veterans-affairs/vets-website/pull/2703/files#diff-a8af7004dab28f2e0d50acf7e7f31b6dR1, just renamed. Maybe don't need this then?

File: src/sass/_hca.scss (hunk 2 above)
Comment (jbalboni) [inline src/sass/_hca.scss]: Hmm, I didn't mean to change anything here, so I'm not sure why it's showing up in the diff.

File: test/hca/components/HealthCareApp.spec.jsx (hunk 3 above)
Comment (jbalboni) [inline test/hca/components/HealthCareApp.spec.jsx]: I think I need to revert any changes here, didn't mean to change any hca stuff.

File: src/js/edu-benefits/utils/options-for-select.js (hunk 1 above)
Comment (jbalboni) [inline src/js/edu-benefits/utils/options-for-select.js]: Yep, I agree. There are a bunch of hca components that could be shared, too.

**Comment by jbalboni** [comment]
@akainic I removed a bunch of files that I did not mean to add. Something about the way I pulled in the last round of changes in the hca branch added a bunch of old files. Fixed now.

**Comment by akainic** [comment]
Looks great! Let's hold off on merging this until https://github.com/department-of-veterans-affairs/vets-website/pull/2703/ is merged, which should be in the next day or two at the latest.

**Comment by jbalboni** [comment]
@akainic I think this is all set now. It's up to date with master and I fixed the linting errors.

File: script/build.js [hunk 4]
```diff
@@ -80,6 +80,7 @@ smith.destination(`../build/${options.buildtype}`);
 const ignoreList = [ 'memorial-benefits/*' ];
 if (options.buildtype === 'production') {
   ignoreList.push('rx/*');
+  ignoreList.push('education/apply-for-education-benefits/apply.md');
```
Comment (akainic) [inline script/build.js]: Should we just have `'education/*'` excluded as Rx does above?

Comment (jbalboni) [inline script/build.js]: There are some existing static pages under education. I think we'd exclude them if I changed it to that.

**Comment by akainic** [comment]
LGTM with one nit question!

File: script/build.js (hunk 4 above)
Comment (akainic) [inline script/build.js]: Gotcha, makes sense!

**Pull request merged**